### PR TITLE
Update hello-indigo.md

### PR DESCRIPTION
**Pull request by paulaylingdev**
Fix a couple of example references which don't work to match [hello-indigo-sbt](https://github.com/PurpleKingdomGames/hello-indigo-sbt/blob/master/src/main/scala/HelloIndigo.scala#L69-L96)

### CODE REVIEW

File: indigodocs/quickstart/hello-indigo.md [hunk 1]
```diff
@@ -397,7 +397,7 @@ The model update function is just a function that has been partially applied wit
 
 In this case, we're interested in two events. A `MouseEvent.Click(x, y)` so that we can add a new dot, and a `FrameTick`. FrameTick is a bit special because it always happens last... and it always happens!
 
-When a mouse click is noticed, we call our `addDot` method with a new Dot, providing the orbital distance and the angle from the center of the screen to the point where we clicked the mouse using `Math.atan2(y, x)`.
+When a mouse click is noticed, we call our `addDot` method with a new Dot, providing the orbital distance and the angle from the center of the screen to the point where we clicked the mouse using `Math.atan2(x, y)`.
```
Comment (davesmith00000) [inline indigodocs/quickstart/hello-indigo.md]: This is an excellent observation, however, `atan2`'s arguments are `y` then `x` (you can check in Javas docs), so this is not a typo I'm afraid.
Why then is the position used in the x,y order in the code you might ask? Try swapping them! It's necessary to have the dot appear under the mouse cursor.
In any case, this change will need to be put back.

**Review by davesmith00000 — CHANGES_REQUESTED** [review]
Thank you very much for updating this! Just one of the changes is incorrect, see my comment, if you could undo it I'll be delighted to merge it.

File: indigodocs/quickstart/hello-indigo.md (hunk 1 above)
Comment (davesmith00000) [reply indigodocs/quickstart/hello-indigo.md]: > This is an excellent observation

I wasn't being sarcastic by the way, I clearly use the arguments the other way around later. Perfectly reasonable to assume it's a typo. :-)

Comment (paulaylingdev) [reply indigodocs/quickstart/hello-indigo.md]: Makes sense! Thanks for the explanation

**Review by davesmith00000 — APPROVED** [review]

**Pull request merged**